Calculator/CalculatorTests/CalculatorTests.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CalculatorTests.swift
+//  CalculatorTests
+//
+//  Created by Whales on 2023/05/31.
+//
+
+import XCTest
+@testable import Calculator
+
+final class CalculatorTests: XCTestCase {
+    var sut: CalculatorItemQueue<Int>!
+    
+    override func setUpWithError() throws {
+        sut = CalculatorItemQueue()
+    }
+    
+    override func tearDownWithError() throws {
+        sut = nil
+    }
+    
+    func test_enqueue함수호출시_배열의원소개수가_늘어나는지() {
+        // given
+        let input = 3
+        let expectation = 1
+        // when
+        sut.enqueue(input)
+        // then
+        XCTAssertEqual(sut.count, expectation)
+    }
+    
+    func test_enqueue함수호출시_배열의원소가_늘어나는지() {
+        // given
+        sut.enqueue(2)
+        let input = 3
+        let expectation = [2, 3]
+        // when
+        sut.enqueue(input)
+        // then
+        XCTAssertEqual(sut.readCalculatorItemQueue(), expectation)
+    }
+    
+    func test_dequeue함수호출시_배열의첫번째원소가_사라지는지() {
+        // given
+        sut.enqueue(2)
+        sut.enqueue(3)
+        sut.enqueue(4)
+        let expectation = [3, 4]
+        // when
+        sut.dequeue()
+        // then
+        XCTAssertEqual(sut.readCalculatorItemQueue(), expectation)
+    }
+    
+    func test_clearQueue함수호출시_배열의원소가_다삭제되면_true를_반환한다() {
```

Comment:
테스트 케이스를 작성하는 방법은 사바사, 회바회기 때문에 정답은 없습니다만, 각 테스트 케이스별 표현 방식은 통일 시켜주시는게 좋겠죠? 
이전 테스트 케이스들은 `~~ 하는지`로 작성하셨으니 이 테스트 케이스 역시 동일하게 작성해주는게 좋습니다!
```suggestion
    func test_clearQueue함수호출시_배열의원소가_다삭제되면_true를_반환하는지() {
```

Reply:
오!! 그런 부분도 통일 시켜주면 좋군요!!
넵넵 수정하고 앞으로도 신경 쓰도록 해보겠습니다!!! 감사합니다 😆